samples/salesforce.gs

```diff
@@ -0,0 +1,81 @@
+// You must create a new connected app in your Salesforce org in order to obtain the CLIENT_ID and CLIENT_SECRET
```

Comment:
Did you mean to commit this as well? Seems somewhat duplicative of the one above, and could use some more polish (commented out code, printing to the log instead of showing information in the spreadsheet).

Reply:
@jordan-cook , could you remove the file `samples/salesforce.gs` in the new commit?
```shell
> git checkout patch-1
> git rm samples/salesforce.gs
> git add .
> git commit .
```

@erickoledadevrel , sorry, it's my first time to support an external fork.